bepasty/tests/conftest.py

```diff
@@ -0,0 +1,34 @@
+from os import close, unlink
+from random import random
+from tempfile import mkstemp
+
+import pytest
+
+from bepasty.app import create_app, create_storage
+
+
+@pytest.fixture(scope='module')
+def testclient(request):
+    '''
+    creates a Flask-testclient instance for bepasty
+    '''
+    app = create_app()
+    db_file, app.config['DATABASE'] = mkstemp()
```

Comment:
The `tmpdir`-fixture is [function-scoped](https://travis-ci.org/Thor77/bepasty-server/jobs/87001685#L230) (maybe there's still a way to use it, but due the lack of experience with py.test I've no idea) and the `tmpdir_factory`-fixture is [not available](https://travis-ci.org/Thor77/bepasty-server/jobs/87002362#L265) for the latest pytest-version for py2.7